app/views/anonymous_feedback/_feedback.html.erb

```diff
@@ -15,6 +15,13 @@
     <br/>
     <strong>comment</strong>: <%= anonymous_contact.details %>
   <% end %>
+<% elsif anonymous_contact.type == "aggregated-service-feedback" %>
+  <strong><%= anonymous_contact.service_satisfaction_rating %> rating</strong>:
+  <% if anonymous_contact.details.present? %>
```

Comment:
If this isn't present, it'll just render an empty string. So you can just get rid of the if block guarding it.